.github/workflows/go-mod-sync.yaml

```diff
@@ -0,0 +1,41 @@
+name: Meshery extensions go.mod sync
+on:
+  push:
+    branches:
+      - "master"
+    paths:
+      - "go.mod"
+jobs:
+  sync:
+    name: sync
+    runs-on: ubuntu-18.04
```

Comment:
@StarTrooper08 @acald-creator we've move to 22, correct?

Reply:
Yes, we had move GitHub actions workflow Ubuntu version specifically to 22.04.
Cause it gives some warnings if we are using 18.04 version or specify latest.

Reply:
Thanks for confirming, @StarTrooper08 :thu

Reply:
Happy to Help😄